modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/operations/util/OperationUtils.java

```diff
@@ -67,7 +66,7 @@ public void updateDeviceOperations(Status status, SyncmlDocument syncmlDocument,
             throws OperationManagementException, DeviceManagementException, NotificationManagementException,
             WindowsOperationException {
 
-        pendingDataOperations = SyncmlUtils.getDeviceManagementService()
+        pendingDataOperations = WindowsAPIUtils.getDeviceManagementService()
                 .getOperationsByDeviceAndStatus(deviceIdentifier, Operation.Status.PENDING);
         if (status.getData().equals(Constants.SyncMLResponseCodes.ACCEPTED) || status.getData().equals
```

Comment:
This equals check should be changed as Constants.SyncMLResponseCodes.ACCEPTED.equals(status.getData()) to avoid null pointer exceptions.